Tests/test_ProtParam.py

```diff
@@ -39,13 +39,23 @@ def test_count_amino_acids(self):
                 self.assertEqual(count_dict[i], self.text.count(i))
 
     def test_get_amino_acids_percent(self):
-        """Calculate amino acid percentages."""
+        """Calculate amino acid percentages (DEPRECATED)."""
```

Comment:
You can put
```
        with self.assertWarns(BiopythonDeprecationWarning):
```
around this, to ensure that the deprecation warning is triggered, but also to avoid showing the deprecation warning on screen when running the test.